libs/usb-drive/package.json

```diff
@@ -15,13 +15,14 @@
     "lint:fix": "pnpm type-check && eslint . --fix",
     "pre-commit": "lint-staged",
     "test": "is-ci test:ci test:watch",
-    "test:ci": "pnpm build && pnpm test:coverage --reporters=default --reporters=jest-junit --maxWorkers=6",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
+    "test:ci": "pnpm build && pnpm test:coverage --run",
+    "test:coverage": "vitest --coverage",
+    "test:watch": "vitest",
     "type-check": "tsc --build"
   },
   "dependencies": {
     "@votingworks/basics": "workspace:*",
+    "@votingworks/fixtures": "workspace:*",
```

Comment:
Added this dependency because I wanted to use the easier tmp file setup/cleanup it provides.